common/reportMetrics.py

```diff
@@ -537,6 +545,8 @@ def replay_sar(metric_file_path, command):
         row_count = 0
         min_timestamp_epoch = 0
         max_timestamp_epoch = -1
+        exclude_tags = agent_config_vars['exclude_tags']
+        logger.debug(exclude_tags)
```

Comment:
Please add more detailed message in the debug mode like "exclude tags: " + str(exclude_tags)